splice_spark/pom.xml

```diff
@@ -19,7 +19,7 @@
         <groupId>com.splicemachine</groupId>
         <version>2.8.0.1845-SNAPSHOT</version>
     </parent>
-    <artifactId>splicemachine-${envClassifier}-${spark.version}_${scala.binary.version}</artifactId>
+    <artifactId>splicemachine</artifactId>
```

Comment:
this artifact name probably now needs changing.  I think calling it 'splicemachine' is flawed. since this will be a change that impacts all users, perhaps we use this opportunity to name the artifact 'splice_spark' ?

Reply:
<dependency>
                    <groupId>com.splicemachine</groupId>
                    <artifactId>splicemachine-cdh5.12.2-2.2.0.cloudera2_2.11</artifactId>
                    <version>${spliceVersion}</version>
                </dependency>
                <dependency>
                    <groupId>com.splicemachine</groupId>
                    <artifactId>hbase_sql-cdh5.12.2</artifactId>
                    <version>${spliceVersion}</version>
                </dependency>
                <dependency>
                    <groupId>com.splicemachine</groupId>
                    <artifactId>hbase_storage-cdh5.12.2</artifactId>
                    <version>${spliceVersion}</version>
                </dependency>
these 3 artifacts is used to make spark adapter work,it is customer-faced,we should make a note so that customers also make changes to their pom when upgrading.
I am good with the changes,spark adapter tests can be easily adapted to this change